test/PublicApiTests/BaseLine/Microsoft.OData.PublicApi.net45.bsl

```diff
@@ -8304,8 +8302,6 @@ public class Microsoft.OData.Client.DataServiceContext {
     public virtual System.Threading.Tasks.Task`1[[Microsoft.OData.Client.DataServiceResponse]] SaveChangesAsync (System.Threading.CancellationToken cancellationToken)
     public virtual System.Threading.Tasks.Task`1[[Microsoft.OData.Client.DataServiceResponse]] SaveChangesAsync (Microsoft.OData.Client.SaveChangesOptions options, System.Threading.CancellationToken cancellationToken)
     public virtual void SetLink (object source, string sourceProperty, object target)
-    public virtual void SetRelatedObject (object source, string sourceProperty, object target)

```

Comment:
Undo this change since it is unrelated to this PR

Reply:
One of previous commit forgot to update the bsl.